src/bidders/index.js

```diff
@@ -0,0 +1,154 @@
+import { context } from '@wikia/ad-engine';
+import { A9 } from './a9/index';
+import { Prebid } from './prebid/index';
+
+
+require('./../../lib/prebid.min');
```

Comment:
Shouldn't it be included in bidders package?

Reply:
It is now after separating bidders to bidders.js package.